frame/society/src/lib.rs

```diff
@@ -161,7 +161,7 @@ decl_storage! {
 		}): Vec<T::AccountId>;
 
 		/// The set of suspended members.
-		pub SuspendedMembers get(fn suspended_member): map T::AccountId => Option<T::BlockNumber>;
+		pub SuspendedMembers get(fn suspended_member): linked_map T::AccountId => Option<T::BlockNumber>;
```

Comment:
we don't use linked maps any more. a `map` should be fine; if you need to enumerate keys from the UI then either store them in the value, or use the `twox_128_concat` hasher.